### PR TITLE
renamed file, and hopefully this works right

### PR DESCRIPTION
**Pull request by ZaCodeHkm**

### CODE REVIEW

File: hakeem_vs_.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Noughts(-1, 0)
+# Crosses(0, 1)
+# .......................................
+# Add subsequent moves below previous moves.
+# ............ Enter Code Below ............
+
+# Noughts: Hakeem
+# Crosses: [Fill in name]
+
+Noughts([1], [0])
```
Comment (nattprog) [inline hakeem_vs_.py]: it should be (1, 0) btw, sry for the confusion xD

Comment (nattprog) [reply hakeem_vs_.py]: @ZaCodeHkm

Comment (ZaCodeHkm) [reply hakeem_vs_.py]: oh understood @nattprog

Comment (nattprog) [reply hakeem_vs_.py]: mm yes such codeness

**Review by nattprog — APPROVED** [review]

**Pull request merged**